### DISCUSSION TITLE
Rest-assured request and response logs not getting in allure report

### DISCUSSION THREAD

**Discussion by prbhosal-tibco** [allure-framework/allure2#2558] [Questions & Support] [answered]
I want to use allure reporting with rest-assured, testng ( TDD ) framework. I followed all the documentation steps I'm able to generate report but in report not able to add request and response block of my request.

**Comment by cheshi-mantu** [comment]
Have you added all the stuff described in the doc?
https://allurereport.org/docs/restassured/#allure-rest-assured
specifically item No 3

**Reply by prbhosal-tibco** [reply, marked as answer]
I have implemented all the steps from docs.
Now it's working fine. I just changed dependency version then it worked.
